Wikipedia/Code/PlacesViewController.swift

```diff
@@ -1736,7 +1736,8 @@ class PlacesViewController: ArticleLocationCollectionViewController, UISearchBar
         }
         switch action {
         case .read:
-            navigate(to: url)
+            let userInfo: [AnyHashable: Any]? = [ArticleSourceUserInfoKeys.articleSource: ArticleSource.places.rawValue]
```

Comment:
This is working when coming from the map. I don't see it working when tapping from the list view though:

![Screenshot 2025-02-06 at 4 21 06 PM](https://github.com/user-attachments/assets/ff27c14e-9ced-45fa-a3c2-2d6efed9ae42)

For both iPad and iPhone. Also just note that iPhone and iPad list views are two separate views, so be sure to test on both.

Reply:
@mazevedofs Still not seeing the source when coming from the iPhone list view (works on iPad though).